login/login_form.php

```diff
@@ -0,0 +1,16 @@
+<link rel="stylesheet" type="text/css" href="login.css">
```

Comment:
no doctype, head or body?

Reply:
did lots of html structure improvements 👍